### DISCUSSION TITLE
Migration of Enterprise github plan to Team based license

### DISCUSSION THREAD

**Discussion by santhamohanrn** [community/community#53082] [Enterprise]
Hi github community team,

We wanted to switch from enterprise github license to a monthly team based license with less users. Please do let me know what is the process to convert to a monthly team based license. I am unable to switch from the existing enterprise billing from the settings options we have. I did not got enough support for the support tickets I have raised. Expecting your suggestion here for this case.

Regards

Santhamohan

**Comment by NJacksonDev** [comment]
Below is an article with instructions on how to change your plan type.

https://docs.github.com/en/billing/managing-billing-for-your-github-account/downgrading-your-github-subscription#downgrading-your-organizations-subscription
